libs/src/services/web3Manager/index.d.ts

```diff
@@ -1,6 +1,7 @@
 class Web3Manager {
   getWalletAddress(): string
   sign(clientChallengeKey: string): Promise<string>
+  ownerWallet: any
```

Comment:
can we type this?

Reply:
would this be string or Wallet from 'eth-wallet' or whatevs?

Reply:
wallet like eth-wallet I believe